ARS/src/main/java/com/CSC450/ars/controller/RatiosCalTest.java

```diff
@@ -0,0 +1,15 @@
+package com.CSC450.ars.controller;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class RatiosCalTest {
+	@Test
+	public void testRatiosCal() {
+		RatiosCal test = new RatiosCal();
```

Comment:
This will have to be:
`AdLocationVisit test_visit = new AdLocationVisit();`
once we remove the unneeded RatiosCal class, and the rest of the test will have to be edited to make it work.